examples/cndp-frr/scripts/cleanup-demo.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) Red Hat Inc.
+
+docker kill client1 client2 cndp-frr1 cndp-frr2
+docker rm client1 client2 cndp-frr1 cndp-frr2
+docker network rm net1 net2 net3
```

Comment:
Should we stop forwarding in this file?

Reply:
The forwarding is inside the container in it's net namespace - it shouldn't leak to the host